api/tests/mocha/controllers/login.spec.js

```diff
@@ -578,16 +580,12 @@ describe('login controller', () => {
       sinon.stub(auth, 'hasAllPermissions').returns(false);
       sinon.stub(auth, 'getUserSettings').resolves({ });
       sinon.stub(config, 'get').returns('fr');
-      const update = sinon.stub(users, 'updateUser').resolves();
       return controller.post(req, res).then(() => {
         chai.expect(cookie.callCount).to.equal(3);
         chai.expect(cookie.args[0][0]).to.equal('AuthSession');
         chai.expect(cookie.args[1][0]).to.equal('userCtx');
         chai.expect(cookie.args[2][0]).to.equal('locale');
         chai.expect(cookie.args[2][1]).to.equal('fr');
-        chai.expect(update.callCount).to.equal(1);
-        chai.expect(update.args[0][0]).to.equal('sharon');
-        chai.expect(update.args[0][1].language).to.equal('fr');
       });
```

Comment:
Please keep the `update.callCount` assertion here, to show that we're not actually updating user-settings.